go.mod

```diff
@@ -293,7 +307,7 @@ replace (
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
 
 	// enforce same SDK and IBC on all dependencies
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.12
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.13
```

Comment:
Question/Reminder, Do we want to keep this script in sync with main/whichever release branch we are using?